server/models/tokens/token.schema.ts

```diff
@@ -0,0 +1,14 @@
+import mongoose, { Schema } from "mongoose";
+const TokenSchema = new Schema({
+    user_id: {
+        type: mongoose.Schema.Types.ObjectId, 
+        ref: 'user'
+    },
+    token: {
+        type: String,
+        required: true,
+    }
+});
+
+
+export default TokenSchema ;
```

Comment:
We could have used a string field on the User model for simplicity instead of separate schema, but this shouldn't complicate things too much.